src/kuberesources.ts

```diff
@@ -9,6 +9,7 @@ export class ResourceKind implements vscode.QuickPickItem {
 }
 
 export const allKinds = {
+    endpoint: new ResourceKind("Endpoiont", "Endpoints", "Endpoint", "endpoints"),
```

Comment:
"Endpoint" is misspelled - first parameter to `ResourceKind`

Reply:
fixed, thanks!